README.md

```diff
@@ -22,3 +22,25 @@ Things you may want to cover:
 * Deployment instructions
 
 * ...
+
+## Developing
+
+You will need the following installed:
+* Ruby 2.6.5
+* PostgreSQL 10
+
+Run the following commands:
+
+```bash
+# Install dependencies
+$ bundler install
+# Set credentials. This will open up an editor. Add the following to the end before closing:
+#   mailer:
+#     gmail_account: gmail@gmail.gmail
+#     password: gmail
+$ rm config/credentials.yml.enc && rails credentials:edit
```

Comment:
@Jess-White This is currently a hack and doesn't have to be necessary depending on what we want to do.

The issue is that when the project was generated the `credentials.yml.enc ` file was generated with private key that is not checked into source control (so I, or anyone else who clones this, doesn't have a way to unlock it). This hack simply regenerates the file locally with a new key with the necessary credentials in it (mailer.gmail_acount and mailer.password). 

It would be better if we could:
* commit the key to source control (only if we don't have sensitive creds in that file)
* use a different strategy for storing these credentials
* update the code that references the credentials and make it not fail if they aren't present (easiest I think)

The references I'm referring to would be:
```ruby
    :user_name            => Rails.application.credentials.mailer[:gmail_account],
    :password             => Rails.application.credentials.mailer[:gmail_app_password],
```
in `config/environments/development.rb`

There might be more references though, would have to check.